lib/report_exporter.rb

```diff
@@ -113,7 +113,8 @@ def self.user_data(user_params)
         "Country",
         "Created at",
         "Updated at",
-        "Country email notifictaions on?",
+        "Local event email notifictaions on?",
+        "Preferred language events email notifictaions on?",
```

Comment:
I would write here "Field of interest events notifications on?" as this is how is called in the frontend. Adding "language" increases confussion as it could be both human language and programming language (and it's both!).
:)